guides/v2.1/cloud/basic-information/starter-develop-deploy-workflow.md

```diff
@@ -78,7 +53,7 @@ The format of the Git clone command is:
 
 The first time you start working in branches for your Starter project, you need to create a Staging branch. This sets up a Staging environment with a code branch matching the Production `master` branch and environment.
 
-Next, create branches from `staging` to develop code, add extensions, and configure 3rd party integrations. Anytime you need to develop custom code, add extensions, integrate with a 3rd party service, work in a develop branch created from `staging`. You will have four active Integration environments available. When you push your an active branch of Git code, one of these Integration environments automatically deploys your code to test.
+Next, create branches from `staging` to develop code, add extensions, and configure 3rd party integrations. Anytime you need to develop custom code, add extensions, integrate with a 3rd party service, work in a develop branch created from `staging`. You can have up to three Active environments available to manage testing and development. When you push an active branch of Git code, one of these Integration environments automatically deploys your code to test.
```

Comment:
Anytime we use "`staging`" or similar, I try to indicate what it is. "the Staging environment" or "the `staging` branch"  because they are two different things...sort of. LOL In fact, environment name is majuscule and the branch is inline code.